packages/fxa-settings/src/models/Account.ts

```diff
@@ -6,12 +6,34 @@ import {
   QueryLazyOptions,
 } from '@apollo/client';
 
+export interface Location {
+  city: string;
+  country: string;
+  state: string;
+  stateCode: string;
+}
+
 export interface Email {
   email: string;
   isPrimary: boolean;
   verified: boolean;
 }
 
+export interface AttachedClient {
+  clientId: string;
+  isCurrentSession: boolean;
+  userAgent: string;
+  deviceType: string;
+  deviceId: string;
+  name: string;
+  lastAccessTime: number;
+  lastAccessTimeFormatted: string;
+  approximateLastAccessTime: number;
+  approximateLastAccessTimeFormatted: string;
+  location: Location;
+  os: string;
+}
+
```

Comment:
Oh bummer, we're duplicating the types from the GraphQL schema? Hmmmmm,
https://www.leighhalliday.com/generating-types-apollo

Reply:
Whaaaat I swear I inquired about this back at the beginning of Settings work. Very good to know!

Reply:
Nice, do we have an issue filed for this? Good to keep in mind for the next maintenance sprint.

Reply:
Filed https://jira.mozilla.com/browse/FXA-2573